src/routes/contract.ts

```diff
@@ -26,87 +27,134 @@ import { NotFoundError } from '../errors';
 import { mismatchedInteractionCount } from '../metrics';
 
 export async function contractHandler(ctx: KoaContext) {
-  const { logger, warp } = ctx.state;
+  const { logger, warp, sortKey, blockHeight } = ctx.state;
```

Comment:
Nit: I'd name sortKey `requestedSortKey` here to further disambiguate it from `evaluatedSortKey`.

Reply:
modifed in [98d2433](https://github.com/ar-io/arns-service/pull/76/commits/98d24339b70bf0a4a4850f269d59164c55794922) and made consistent